docs/How-to-run-blackbox-testing-against-snaps.md

```diff
@@ -0,0 +1,79 @@
+# Blackbox Testing against Edgexfoundry Snap
+
+Helpers scripts have been created to support blackbox testing against Edgexfoundry snap. The scripts will install the appropriate snaps and enable required services for testing.
+
+Once the helper scripts have been executed the tester must then run two additional commands that will execute the actual blackbox tests.
+
+
+Step 1: Setup environment variable
+
+Locate the env.sh file at the following path
+
+**/bin/env.sh**
+
+Uncomment the following environment variable
+
+**export TEST_ENV=localhost**
+
+If you wish to use different channels for edgexfoundry and/or app-service-configurable then add the following environment variable
+
+**export EF_CHANNEL=stable**
+
+Accepted values for EF_CHANNEL are stable, edge, candidate
+
+**export ASC_CHANNEL=stable**
+
+Accepted values for ASC_CHANNEL are stable, edge, candidate
+
+You can also use locally built snaps of either edgexfoundry and/or app-service-configurable by specifying the following environment variables
+
+**export EF_SNAP=path_to_snap**
+
+**export ASC_SNAP=path_to_snap**
+
+Replace path_to_snap with an absolute path to each snap respectively.
+
+Note, if both ASC_CHANNEL and ASC_SNAP variables are exported then ASC_SNAP will take precendence and ASC_CHANNEL will be ignored. This is also true for EF_SNAP and EF_CHANNEL.
+
+If you wish to run the tests with security than set the following environment variable:
+
+**export SECURITY_SERVICE_NEEDED=true**
+
+Once you have set the environment variables, save the file. Navigate back to the root directory of the project and run the following bash command
+
+**. bin/env.sh**
+
+Step 2: Deploy test environment
+
+Navigate to the following directory
+
+**cd misc/scripts/**
```

Comment:
I agree with Tony. The `deploy-edgeX-snaps.sh` should be at the top-level. 
The rest of scripts in misc are not necessary since you can modify the `bin/setupSecurityserviceTest.sh`, `bin/cleanSecurityserviceTest.sh`, and `bin/security/setupSecurityAccount.sh` to do the same thing.

Reply:
File moved to top level and all other scripts in misc/scripts directory has been removed. Both misc and scripts directories have also been removed.